runbotics/runbotics-orchestrator-ui/src/src-app/components/tables/HistoryTable/HistoryTable.tsx

```diff
@@ -42,24 +42,52 @@ const HistoryTable = forwardRef<any, HistoryTableProps>(({ botId, processId, sx,
     const { firstValueFrom } = useQuery();
     const pageFromUrl = firstValueFrom('page');
     const pageSizeFromUrl = firstValueFrom('pageSize');
+    const processInstanceIdFromUrl = firstValueFrom('instanceId');
 
     const [panelInfoState, setPanelInfoState] = useState<PanelInfoState>({ show: false });
     const [page, setPage] = useState(pageFromUrl ? parseInt(pageFromUrl, 10) : 0);
     const [pageSize, setPageSize] = useState(pageSizeFromUrl ? parseInt(pageSizeFromUrl, 10) : DefaultPageSize.TABLE);
+    const [instanceId, setInstanceId] = useState(processInstanceIdFromUrl);
```

Comment:
imo useState(processInstanceIdFromUrl ?? null);

because when you clear this value you set instanceId to null but if processInstanceIdFromUrl would be not specified it would be undefined